test/test_regression_cli.py

```diff
@@ -46,12 +46,13 @@ def get_csv_md5(dataset, engines, tmpdir):
 
 def setup_module():
     """Update retriever scripts and cd to test directory to find data"""
-    os.chdir("./test/")
+    os.chdir(os.path.dirname(os.path.abspath(__file__)))
     os.system("retriever update")
 
 
 def teardown_module():
     """Cleanup temporary output files after testing and return to root directory"""
+    os.chdir(os.path.dirname(os.path.abspath(__file__)))
```

Comment:
make sure you are in the test directory and then remove all the files not needed.
Note: according to the test design `raw_data/AvianBodySize`  and `test/raw_data/DelMoral2010` should not be deleted